frontend/pages/status/index.js

```diff
@@ -0,0 +1,279 @@
+		};
+	}, []);
+
+	useLayoutEffect(() => {
+		const imageLoader2880 = new Image();
+		imageLoader2880.src = `${AWS_ASSETS_PATH}/product-background-2880x1620.png`;
+		imageLoader2880.onload = () => {
+			setBackgroundLoaded2880(true);
+		};
+	}, []);
+
+	useLayoutEffect(() => {
+		const imageLoader3840 = new Image();
+		imageLoader3840.src = `${AWS_ASSETS_PATH}/product-background-3840x2160.png`;
+		imageLoader3840.onload = () => {
+			setBackgroundLoaded3840(true);
+		};
+	}, []);
+
+	useLayoutEffect(() => {
+		if (apiServerData?.data.status == "ok") {
+			setAPIServerDataStatus({
+				color: healthyStatusColor,
+				text: healthyStatusText
+			});
+		}
+		if (crawlersServerData?.data.status == "ok") {
+			setCrawlersServerDataStatus({
+				color: healthyStatusColor,
+				text: healthyStatusText
+			});
+		}
+		if (gethData?.data?.current_block) {
+			setGethDataStatus(gethData?.data?.current_block);
+		}
+		if (crawlersData?.data) {
+			if (crawlersData?.data?.ethereum_txpool_timestamp) {
+				setTxpoolDataStatus(
+					crawlersData?.data?.ethereum_txpool_timestamp
+						.replace(/^.+T/, "")
+						.replace(/\..+/, "")
+				);
+			}
+			if (crawlersData?.data?.ethereum_trending_timestamp) {
+				setTrendingDataStatus(
+					crawlersData?.data?.ethereum_trending_timestamp
+						.replace(/^.+T/, "")
+						.replace(/\..+/, "")
+				);
+			}
+		}
+
+		if (dbServerData?.data?.status == "ok") {
+			setDBServerDataStatus({
+				color: healthyStatusColor,
+				text: healthyStatusText
+			});
+		}
+		if (latestBlockDBData?.data?.block_number) {
+			setLatestBlockDBDataStatus(latestBlockDBData?.data?.block_number);
+		}
+	}, [apiServerData, crawlersServerData, dbServerData]);
+
+	const margin = useBreakpointValue({
+		base: "1%",
+		sm: "2%",
+		md: "3%",
+		lg: "15%",
+		xl: "20%",
+		"2xl": "25%"
+	});
+
+	return (
+		<Flex
+			bgPos="bottom"
+			bgColor="transparent"
+			backgroundImage={`url(${assets[`${background}`]})`}
+			bgSize="cover"
+			// boxSize="full"
+			minH="100vh"
+			direction="column"
+			alignItems="center"
+			pb={24}
+		>
+			<Stack mx={margin} my={12} maxW="1700px">
+				<Heading
+					as="h2"
+					size="md"
+					placeSelf="center"
+					px={12}
+					py={2}
+					borderTopRadius="xl"
+				>
+					{`Status page`}
+				</Heading>
+				<chakra.span pl={2} px={12} py={2} width="400px">
+					<Flex mb={3}>
+						<Text>Backend server</Text>
+						<Spacer />
+						<Text color={apiServerDataStatus.color}>
+							{apiServerDataStatus.text}
+						</Text>
+					</Flex>
+					<br />
+					<Flex mb={3}>
+						<Text>Crawlers server</Text>
+						<Spacer />
+						<Text color={crawlersServerDataStatus.color}>
+							{crawlersServerDataStatus.text}
+						</Text>
+					</Flex>
+					<Flex mb={3}>
+						<Text>Latest block in Geth</Text>
+						<Spacer />
+						<Text>{gethDataStatus}</Text>
+					</Flex>
+					<Flex mb={3}>
+						<Text>Txpool latest record ts</Text>
+						<Spacer />
+						<Text>{txpoolDataStatus}</Text>
```

Comment:
Its Good idea to provide page with `isLoading` indicators for each fetchable data and use Skeleton

```js
<Skeleton  isLoaded={!isLoading} > <Text>{gethDataStatus}</Text> </Skeleton>
```

^^^ skeleton component might need to know height of text block before data comes in. If your skeletons do not appear, try modifying `h` prop for `<Text>` or `<Skeleton>`